src/Template/Pages/UserProfile/view.twig

```diff
@@ -0,0 +1,55 @@
+{% do _view.assign('title', 'User Profile') %}
+
+<div class="userprofiles-view">
+
+    {% if object.attributes.username %}
+    <header>
+        <h1>{{ object.attributes.username }}</h1>
+    </header>
+    {% endif %}
+
+    <div class="profile-form">
+
+        {{ Form.create({'defaults': object.attributes, 'schema': {}}, {
+            'url': {'_name': 'profile:save', 'object_type': 'users'},
+            'id': 'form-main',
+        })|raw }}
+
+            <div class="form-column">
+                <section class="fieldset order-0">
+                    <div class="tab-container">
+                        {{ Form.hidden('id', {'value': object.id})|raw }}
+                    </div>
+                </section>
+
+                {% for key,group in Site.profileGroups() %}
+                <section class="fieldset order-{{key + 1}}">
+                    {% if key != '0' %}
+                    <header>
+                        <h2>{{ __(group) }}</h2>
+                    </header>
+                    {% endif %}
+                    <div class="tab-container">
+                        {% set jsonKeys = [] %}
+                        {% set attributes = Site.profileFields(object.attributes, group) %}
+                        {% for key, value in attributes %}
+                            {% set options = Schema.controlOptions(key, value, schema.properties[key]) %}
```

Comment:
Is there a `schema` loaded via controller?

Reply:
Nope. I add some logic in controller, like
```
$revision = Hash::get($response, 'meta.schema.users.revision', null);
$schema = $this->Schema->getSchema('users', $revision);
$this->set('schema', $schema);
```